rest/route/middleware.go

```diff
@@ -531,7 +438,7 @@ func (m *CommitQueueItemOwnerMiddleware) ServeHTTP(rw http.ResponseWriter, r *ht
 		if user.Id != *patch.Author {
 			gimlet.WriteResponse(rw, gimlet.MakeJSONErrorResponder(gimlet.ErrorResponse{
 				StatusCode: http.StatusUnauthorized,
-				Message:    "Not authorized",
+				Message:    "Not authorized patch author",
```

Comment:
This wording confuses me a bit, because it's not the patch author that's not authorized (whoever is hitting this is specifically not the patch author). So it's technically `not authorized because not patch author or person with the proper permissions`.